components/Modal.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { TouchableWithoutFeedback, View } from 'react-native';
+import { BottomSheet, ListItem } from 'react-native-elements';
+import { styles } from './Modal.styles';
+
+export const Modal = (props) => {
+  return (
+    <BottomSheet
+      isVisible={props.isVisible}
+      containerStyle={styles.backdrop}
+    >
+      <View style={styles.list}>
+        <ListItem
+          Component={TouchableWithoutFeedback}
+          bottomDivider={true}
+          containerStyle={styles.modalText}
+        >
+          <ListItem.Content style={styles.modalContent}>
```

Comment:
given the listitems have some things in common you could make a subcomponent something like `ModalItem`. Depending on if its not to much of conditions to set the right style classes.

Reply:
so if that is to much for now. leave it as is (because its working fine!)

Reply:
Hm, they share nearly no styling. But might still be more readable. Will try it!